halo2_proofs/Cargo.toml

```diff
@@ -80,15 +80,14 @@ serde_json = "1"
 getrandom = { version = "0.2", features = ["js"] }
 
 [features]
-default = ["batch","bits"]
+default = ["batch"]
```

Comment:
Why are removing the `bits` feature? We should bring it back even if it's not as `default`.